src/app/home/home.component.html

```diff
@@ -14,6 +14,7 @@
   ~    limitations under the License.
   -->
 
+<app-logged-in-banner *ngIf="user$ | async"></app-logged-in-banner>
```

Comment:
I think the intent is to use this component on a new page that doesn't exist yet. I know you need this for testing, but I don't think I'd commit this.

Reply:
Sure. Removed that line so now the component isn't being used anywhere